File: basics/Q59/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var (
		a float64
		b float64
	)
	fmt.Print("Enter a: ")
	fmt.Scan(&a)
	fmt.Print("Enter b: ")
	fmt.Scan(&b)
	y := a * math.Sin(b)
	fmt.Println("y is", y)
}

File: basics/Q59/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter a:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let a: f64 = input.trim().parse().expect("Not a valid number");

    input.clear();

    println!("Enter b:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let b: f64 = input.trim().parse().expect("Not a valid number");

    let y = a * b.sin();

    println!("y is {}", y);
}